modules/odc-access-web-terminal.adoc

```diff
@@ -19,14 +19,25 @@ The web terminal remains open until you close it or until you close the browser
 
 . To launch the web terminal, click the command line terminal icon (image:odc-wto-icon.png[title="wto icon"]) in the masthead of the console. A web terminal instance is displayed in the *Command line terminal* pane. This instance is automatically logged in with your credentials.
 
-. Select the project where the `DevWorkspace` CR must be created from the *Project* drop-down list. By default, the current project is selected.
+. If a project has not been selected in the current session, select the project where the `DevWorkspace` CR must be created from the *Project* drop-down list. By default, the current project is selected.
 +
 [NOTE]
 ====
-* The `DevWorkspace` CR is created only if it does not already exist.
+* The `DevWorkspace` CR is created only if it does not already exist. The {web-terminal-op} installs the DevWorkspace Operator as a dependency.
+
 * The `openshift-terminal` project is the default project used for cluster administrators. They do not have the option to choose another project.
 ====
 
-. Click *Start* to initialize the web terminal using the selected project. After the web terminal is initialized, you can use the preinstalled CLI tools like `oc`, `kubectl`, `odo`, `kn`, `tkn`, `helm`, `kubens`, `subctl`, and `kubectx` in the web terminal.
+ifndef::openshift-rosa,openshift-dedicated[]
+. Optional: Set the web terminal timeout for the current session:
+.. Click Timeout.
+.. From the drop-down list, select a timeout interval of Seconds, Minutes, Hours, or Milliseconds.
+
+. Optional: Select a custom image for the web terminal to use.
+.. Click Image. 
+.. Enter the URL of the image that you want to use.
+endif::openshift-rosa,openshift-dedicated[]
+
+. Click *Start* to initialize the web terminal using the selected project. 
```

Comment:
*PEER REVIEW* Moved this sentence unchanged from [line 30](https://github.com/openshift/openshift-docs/pull/67647/files#diff-3280d02527d34051c6d259501fa40e60cdf0fc555e2183f992e78da26dcf2fffL30).